src/settings-ui/Settings.UI/SettingsXAML/Views/CmdNotFoundPage.xaml

```diff
@@ -19,37 +26,86 @@
+                    <controls:SettingsCard x:Uid="CmdNotFound_WinGetClientDetection">
+                        <StackPanel Orientation="Horizontal" Spacing="8">
+                            <SymbolIcon
+                                Foreground="Green"
+                                Symbol="Accept"
+                                Visibility="{x:Bind Mode=OneWay, Path=ViewModel.IsWinGetClientModuleDetected, Converter={StaticResource BoolToVisibilityConverter}}" />
+                            <SymbolIcon
+                                Foreground="Red"
+                                Symbol="Cancel"
+                                Visibility="{x:Bind Mode=OneWay, Path=ViewModel.IsWinGetClientModuleDetected, Converter={StaticResource BoolToInvertedVisibilityConverter}}" />
+                            <Button
+                                x:Uid="CmdNotFound_InstallButton"
+                                Command="{x:Bind ViewModel.InstallWinGetClientModuleEventHandler}"
+                                IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.IsWinGetClientModuleDetected, Converter={StaticResource BoolNegationConverter}}"
+                                Style="{StaticResource AccentButtonStyle}" />
+                        </StackPanel>
+                    </controls:SettingsCard>
+                </custom:SettingsGroup>
+                <custom:SettingsGroup x:Uid="CmdNotFound_InstallationGroup" IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabledGpoConfigured, Converter={StaticResource BoolNegationConverter}}">
+                    <controls:SettingsCard
+                        x:Uid="CmdNotFound_Enable"
+                        HeaderIcon="{ui:BitmapIcon Source=/Assets/Settings/FluentIcons/FluentIconsCmdNotFound.png}"
+                        IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabledGpoConfigured, Converter={StaticResource BoolNegationConverter}}">
+                        <StackPanel Orientation="Horizontal" Spacing="8">
+                            <SymbolIcon
+                                Foreground="Green"
+                                Symbol="Accept"
+                                Visibility="{x:Bind Mode=OneWay, Path=ViewModel.IsCommandNotFoundModuleInstalled, Converter={StaticResource BoolToVisibilityConverter}}" />
+                            <SymbolIcon
+                                Foreground="Red"
+                                Symbol="Cancel"
+                                Visibility="{x:Bind Mode=OneWay, Path=ViewModel.IsCommandNotFoundModuleInstalled, Converter={StaticResource BoolToInvertedVisibilityConverter}}" />
+                            <Button
+                                x:Uid="CmdNotFound_InstallButton"
+                                Command="{x:Bind ViewModel.InstallModuleEventHandler}"
+                                IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.IsCommandNotFoundModuleInstalled, Converter={StaticResource BoolNegationConverter}}"
```

Comment:
Bettet a visibility property for both buttons?

Reply:
Can give it a try. We have visibility stuff already in there for the others.